### ISSUE TITLE
Access to certain job areas if no positions filled.

### ISSUE THREAD

**Issue by ghost** [d3athrow/vgstation13#32496]
[featurerequest][discussion]
Idea would involve, for example, normal medical doctors being allowed into the CMO office while there is none on station; or possibly security into the bridge and captain's office if no heads are on station, for a more extreme example.
To what level should I do this?

**Comment by Servalfang** [comment]
isnt this already a mechanic we made deprecated

**Comment by ghost** [comment]
> isnt this already a mechanic we made deprecated

When and where?

**Comment by DorainGrey** [comment]
I think there's limited scope for this, security/heads getting access to captains office in absence of a captain would be good as it just saves the effort of breaking in.

I don't think there's any reason however for enabling access to other heads offices by the department staff, there's nothing there that's essential to operating any department AFAIK. If you want the goodies, play head roles or find a way to break in yourself.

The one exception to this would be the CE with opening secure storage, however a better solution to that issue is just leaving the shutters default open, like is done on some of our maps roidstation etc

**Comment by Kurfursten** [comment]
Better solution: put two sets of shutters over the storage, CE one defaults open and normal engie one defaults closed. Engineers can keep the dangerous stuff secure but CE can override if someone steals access.

> mode already depreciated

Kinda. There's a button under Secrets that allows Egalitarian Station, which removes much access to the station. It's not supposed to open security or heads of staff rooms. It's extremely inconsistent however and some doors that have multiple accesses like botany stay locked while some doors like CMO office lose access inappropriately. It's rarely used so no one fixed it.

**Comment by Cartlord** [comment]
Maybe I've just gone insane, but, don't we have a feature that gives people bonus access on lowpop?

**Comment by Kurfursten** [comment]
Nope, we really don't. A lot of people tend to hand out access on lowpop though.

**Comment by Dacendeth** [comment]
There's that thing admins can turn on im pretty sure before the round starts that lets anyone into any non command/sec area.
